packages/manager/apps/key-management-service/src/pages/dashboard/tabs/GeneralInformations.tsx

```diff
@@ -26,12 +26,12 @@ function GeneralInformationsTab() {
   }
 
   return (
-    <div className="grid xs:grid-cols-1 sm:grid-cols-3 py-6">
+    <div className="grid xs:grid-cols-1 sm:grid-cols-2 md:grid-cols-3 py-6">
       <div className="p-3">
         <InformationsTile okmsData={data.data} />
       </div>
       <div className="p-3">
```

Comment:
```suggestion
    <div className="grid xs:grid-cols-1 sm:grid-cols-2 md:grid-cols-3 py-6 gap-3">
        <InformationsTile okmsData={data.data} />
```
use gap property, not sure which tailwind value to choose.